cmd/editorconfig-checker/main.go

```diff
@@ -78,6 +88,11 @@ func parseArguments() {
 	cmdlineConfig = config.Config{}
 	writeConfigFile = false
 
+	// check the NO_COLOR environment variable before parsing the arguments, so the arguments can override
+	if nocolor := os.Getenv("NO_COLOR"); nocolor != "" && nocolor != "0" && !strings.EqualFold(nocolor, "false") {
```

Comment:
You should consider using strconv.ParseBool it supports true,/false,0/1, t/f

Reply:
good point, that makes that `if` a little bigger, but takes out a lot of guesswork.